wasm.md

```diff
@@ -64,7 +64,8 @@ WebAssembly instructions are space-separated lists of instructions.
     syntax Instrs ::= List{Instr, ""} [klabel(listInstr)]
  // -----------------------------------------------------
     rule <k> .Instrs           => .       ... </k>
-    rule <k> I:Instr IS:Instrs => I ~> IS ... </k>
+    rule <k> I:Instr .Instrs   => I       ... </k>
```

Comment:
What's the point of (re)introducing these rules?

Reply:
Having the two rules (one which explicitly handles the `.Instrs` case) means that we won't generate a bunch of extra `.Instrs` in the tail of the instruction stack.